simtools/ray_tracing/ray_tracing.py

```diff
@@ -459,29 +443,12 @@ def _analyze_image(
         tuple
             Tuple containing analyzed results.
         """
-        if use_rx:
-            containment_diameter_cm, centroid_x, centroid_y, eff_area = self._process_rx(
-                photons_file
-            )
-            containment_diameter_deg = containment_diameter_cm * cm_to_deg
-            image.set_psf(containment_diameter_cm, fraction=containment_fraction, unit="cm")
-            image.centroid_x = centroid_x
-            image.centroid_y = centroid_y
-            eff_area = eff_area * tel_transmission
-            image.set_effective_area(eff_area)
-        else:
-            containment_diameter_cm = image.get_psf(containment_fraction, "cm")
-            containment_diameter_deg = image.get_psf(containment_fraction, "deg")
-            centroid_x = image.centroid_x
-            eff_area = image.get_effective_area() * tel_transmission
-
-        eff_flen = np.nan if this_off_axis == 0 else centroid_x / tan(this_off_axis * pi / 180.0)
         return (
             this_off_axis * u.deg,
-            containment_diameter_cm * u.cm,
-            containment_diameter_deg * u.deg,
-            eff_area * u.m * u.m,
-            eff_flen * u.cm,
+            image.get_psf(containment_fraction, "cm") * u.cm,
+            image.get_psf(containment_fraction, "deg") * u.deg,
+            image.get_effective_area(tel_transmission) * u.m * u.m,
+            np.nan if this_off_axis == 0 else image.centroid_x / tan(this_off_axis * pi / 180.0),
```

Comment:
We should also use centroid_y at one point for the effective focal length: See #981

Reply:
Yes, agree. But not as part of this PR.